docs/make.jl

```diff
@@ -17,4 +17,4 @@ makedocs(
     ],
 )
 
-deploydocs(repo = "github.com/domluna/JuliaFormatter.jl.git")
+deploydocs(repo = "github.com/domluna/JuliaFormatter.jl.git", push_preview = true)
```

Comment:
um guessing from the error, maybe this change needs to be done with the write access to the repository ?
@domluna this would allow us to preview the documentation of this package without local build, so I would like to just let you know this, but I will erase this from the commit history later ;) (ref: https://discourse.julialang.org/t/ann-documenter-0-24/31381)

Reply:
makes sense :+1: